### PR TITLE
📦 Update Kotlin, Compose and other dependencies

### LINKED ISSUE #17: 📦 Update dependencies
## Description
Primary update Gradle plugin, kotlin, coroutines and compose

### PR DESCRIPTION
**Pull request by semanticer**
## Summary
Just a routine dependency update 

## Relationships

Closes #17

### CODE REVIEW

File: komposable-architecture/src/test/java/com/toggl/komposable/StoreSubscriptionTests.kt [hunk 1]
```diff
@@ -4,22 +4,25 @@ import com.toggl.komposable.common.StoreCoroutineTest
...
         coVerify(exactly = 1) {
             testSubscription.subscribe(any())
         }
     }
 
     @Test
-    fun `all actions coming from a subscription are reduced in correct order`() = runBlockingTest {
+    fun `all actions coming from a subscription are reduced in correct order`() = runTest {
         testSubscription.stateFlow.value = TestAction.DoNothingAction
+        runCurrent()
```
Comment (daividssilverio) [inline komposable-architecture/src/test/java/com/toggl/komposable/StoreSubscriptionTests.kt:23]: oh, new cool stuff

**Review by daividssilverio — APPROVED** [review]
Looks pretty good.
Not sure if it's my PC, the emulator or the sample app itself... but the sample is running quite slow for me (on main as well).

**Comment by semanticer** [comment]
I re-tested the speed just to be sure and everything seems to be fine on my PC/emulator 🤷‍♂️

**Comment by daividssilverio** [comment]
It turns out it was my setup. It was slow all day, not quite sure why.

**Pull request merged**